packages/common/src/ContractInteractor.ts

```diff
@@ -458,7 +467,7 @@ export class ContractInteractor {
           method: 'eth_call',
           params: [
             {
-              from: relayCallABIData.relayRequest.relayData.relayWorker,
+              from,
```

Comment:
some networks fail the CALL request if caller doesn't have enough to cover the gas price...
which means that in dry-run, we must have `gasPrice:0`
This, however, will alter our gas-price calculations
(BTW: this is, by itself, a good reason to have the ZERO_ADDRESS as a "DRY_RUN_ADDRESS", since zero address has balance on all networks...

Reply:
I can make the dry-run origin address configurable. I assume some rollups won't have any Ether on address(0). What do you think?